toolchain/check/import_ref.cpp

```diff
@@ -1265,20 +1266,58 @@ class ImportRefResolver {
   llvm::SmallVector<Work> work_stack_;
 };
 
+// Returns a pair of ImportIRInsts equivalent to the ImportRef currently being
```

Comment:
```suggestion
// Returns a list of ImportIRInsts equivalent to the ImportRef currently being
```
(To me the old comment suggested that this would return two (a pair of) `ImportIRInst`s.)

Reply:
Done